### ISSUE TITLE
vue.config.js Example and Documentation is Impossible to Find

### ISSUE THREAD

**Issue by jacksonrdlc** [vuejs/vue-cli#1581]
### What problem does this feature solve?
I use to be able to find the vue.config.js example layout and documentation but now I can't find it anywhere. There should be link on the main readme of the cli. This is a super important file created in the vue-cli@3 with very little examples around the internet because it is so new. This will help developers understand how to set up their config and have more people testing the beta.

### What does the proposed API look like?
a link to the vue.config.js example page and maybe more depth into the example.

**Comment by yyx990803** [comment]
Go to the website, search for vue.config.js.

**Comment by checktravis** [comment]
TL;DR: most likely replacement: https://github.com/vuejs/vue-cli/blob/dev/docs/config/README.md 

I agree the previous example vue.config.js was a bit buried. I had it bookmarked and when that bookmark broke, I reviewed the changelogs and tracked through the commit history to figure out what happened to it.  I found the above updated readme which is much more detailed (nice work 👍 ) but I miss the example vue.config.js as a helpful starting point.

**Comment by gnicol-bzh** [comment]
Hello

Maybe a little off topic: while following [Debugging in VS Code and Chrome](https://vuejs.org/v2/cookbook/debugging-in-vscode.html), I used Vue CLI to create a default "hello world" app. Then they talk about `config/index.js` and `vue.config.js` but I was unable to find any in my new project.

It may be ovious for Vue veterans, but I'm quite new to Vue & Webpack and it could usefull to quickly mention the link given by @checktravis in this cookbook ;).

Now I have to search for `config.index.js`... (but now it's really off topic)

**Comment by JuanCarlosC** [comment]
I was scratching my head for over an hour about this and I realized that I created the `vue.config.js` file within my `/src` directory. I just moved it outside next to the `package.json` file (just like the docs say), and everything worked.
